src/content/en/2021/media.md

```diff
@@ -1,20 +1,875 @@
+  )
+}}
+
+### Video attributes
+
+The video HTML5 tag has a number of attributes that can be used to define how the video player will appear to end users.
+
+Let's look at the most common attributes and how they are used inside the `<video>` tag:
+
+{{ figure_markup(
+  image="video-element-attributes.png",
+  caption="Video element attributes.",
+  description="A bar chart showing the number of times various attributes on the HTML video element were found, on both desktop and mobile. With the exception of `controls`, which is slightly more common on mobile, every attribute is more common on desktop by 5-30%. The tallest pair of bars is `preload`; `buffered`s are the shortest. None of the bars are labelled, but their desktop numbers are approximately: `autoplay`: 100,000; `buffered`: 0; `controls`: 45,000; `controllist`: 1,000; `crossorigin`: 30,000; `height`: 3,000; `loop`: 50,000; `muted`: 55,000; `playsinline`: 52,000; `poster`: 24,000; `preload`: 115,000; `src`: 90,000; width: 60,000.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=1099468813&format=interactive",
+  sheets_gid="1827909463",
+  sql_file="video_attribute_count_mobile.sql"
+  )
+}}
+
+#### Preload
+
+The most commonly used attribute is preload. The preload attribute gives the browser a hint on the best way to handle the video download. There are four possible options: `auto`, `metadata`, `none`, and an empty response = `auto`.
+
+{{ figure_markup(
+  image="video-preload-values.png",
+  caption="Video preload values.",
+  description="A bar chart showing the prevalence of the various preload attribute values, on desktop and mobile, in both 2020 and 2021. `auto` was used in 36% of preload attributes in 2020 on desktop, falling to 23% in 2021. On mobile, `auto` fell from 27% to 18%. `metadata` was used in 24% of preload attribuets on desktop in 2020, rising slightly to 25% in 2021. On mobile, `metadata` usage rose from 33% to 36%. The `none` value was present in 33% of preload attributes on desktop in 2020, rising to 46% in 2021. On mobile, usage also rose, from 33% to 40%. Finally, around 4% of preload attributes were present but left blank on desktop, and 5% on mobile, with no change for either between 2020 and 2021.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=1322480946&format=interactive",
+  sheets_gid="7133313",
+  sql_file="video_attribute_count_mobile.sql"
+  )
+}}
+
+Interestingly, we see a large push away from `preload=auto` on both mobile and desktop, with most of the movement to `preload=none.`  While it is possible that this changed for many videos, it could just be that the new videos added to the web over the last year utilize the "none" parameter more than in the past. From a page weight perspective—this is a large win for the web.
```

Comment:
```suggestion
Interestingly, we see a large push away from `preload=auto` on both mobile and desktop, with most of the movement to `preload=none.`  While it is possible that this changed for many videos, it could just be that the new videos added to the web over the last year utilize the `none` parameter more than in the past. From a page weight perspective—this is a large win for the web.
```

Reply:
Applied